src/components/Workspace.js

```diff
@@ -437,71 +437,20 @@ class Workspace extends Component {
     this.updateCode();
   }
 
-  remix = () => {
-    const {
-      code,
-      changeReadOnly,
-      createProgram,
-      fetchProgram,
-      saveProgram,
-    } = this.props;
-
-    // Need to fetch the program again since the current code has editable="false" tags
-    return Promise.all([fetchProgram(code.id), createProgram(code.name)])
-      .then(([fetchData, createData]) => {
-        logger.log(JSON.stringify({
-          event: 'remix', userId: createData.user_id, sourceProgramId: code.id, newProgramId: createData.value.id,
-        }));
-        return saveProgram(createData.value.id, fetchData.value.content,
-          createData.value.name, fetchData.value.reference_of);
-      })
-      .then(() => {
-        changeReadOnly(false);
-        this.createWorkspace();
-      });
-  }
 
   render() {
     const { children, code, rover } = this.props;
 
     return (
-      <Box m={1}>
```

Comment:
Removing this padding allowed me to make the workspace line up with the program title bar without also adding the same padding to the title bar, which it didn't really need.
I don't think it makes the workspace look too tight, but you can let me know what you think.